config/application.rb

```diff
@@ -14,6 +14,9 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.    
+    # the framework and any gems in your application.
+    config.time_zone = 'Europe/Moscow'
```

Comment:
```suggestion
    config.time_zone = 'Moscow'
```
можно просто Москва указать
список доступных таймзон можно посмотреть тут: https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html (в константе MAPPING)

Reply:
Не был уверен, нужно ли использовать ключи или значения из этого хэша. Получается без разницы, понятно.